src/oscar/apps/order/abstract_models.py

```diff
@@ -223,24 +223,23 @@ def shipping_tax(self):
 
     @property
     def shipping_status(self):
-        events = self.shipping_events.all()
+        events = self.shipping_events.order_by('-date_created').all()
```

Comment:
The [Django docs](https://docs.djangoproject.com/en/dev/topics/db/models/#automatic-primary-key-fields) aren't very explicit about it, but still leave me with the impression that the primary key will only ever go up. "Auto increment" suggests the same, doesn't it?